modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc

```diff
@@ -0,0 +1,102 @@
+= Migrating User Key Encryption to Master Key Encryption
+
+If you need to migrate from User Key-based to Master Key-based encryption, there are several steps that you need to follow to ensure a smooth and complete transition. 
+These are:
+
+. xref:disable-user-key-based-encryption[Disable User Key-based encryption]
+. xref:remove-the-encryption-records-from-the-owncloud-database[Remove the encryption records from the ownCloud database]
+. xref:remove-the-`files_encryption`-directory[Remove the `files_encryption` folder]
+. xref:encrypt-the-filesystem-using-master-key-based-encryption[Encrypt the filesystem using Master Key-based encryption]
+
+include::{partialsdir}/configuration/server/enable-single-user-mode.adoc[leveloffset=+1]
+
+[[disable-user-key-based-encryption]]
+== Disable User Key-based Encryption
+
+The first part of the migration process is to take is to decrypt all files and to disable encryption in ownCloud, which requires three commands to be executed. 
+These commands are: xref:configuration/server/occ_command.adoc#encryption[`occ encryption:decrypt-all` and `occ encryption:disable`], and xref::configuration/server/occ_command.adoc#apps-commands[`occ app:disable`].
```

Comment:
The commands shown are in a line (like a sentence).
What about making that like a list underneath (not necessarily a bullet list, but at least underneath).
This would improve readbility.

Reply:
I started off doing it this way, but it read better (a purely subjective opinion) as part of a sentence.